frontend/src/lib/components/report/elements/SliceElement.svelte

```diff
@@ -61,23 +61,25 @@
 
 {#if sliceElementOptions !== undefined && sliceElementSpec && table}
 	<div class="w-full">
-		<div class="flex items-center mb-2">
-			<h3 class="text-lg">
-				Slice <span class="font-semibold">{sliceElementOptions.sliceName} </span>
-				{#if sliceElementSpec.modelName}
-					model
-					<span class="font-semibold">{sliceElementSpec.modelName}</span>
-				{/if}
-			</h3>
-			<Button
-				variant="outlined"
-				class="ml-4"
-				on:click={() =>
-					goto(
-						`/project/${sliceElementOptions?.project.ownerName}/${sliceElementOptions?.project.name}` +
-							encodeParams()
-					)}>Explore</Button
-			>
+		<div class="flex items-center mb-2 justify-between">
```

Comment:
I think I already fixed this in my PR. Don't think this extra div is needed.

Reply:
Oop yeah hadn't pulled